jupyter/utils/addons/apply.sh

```diff
@@ -31,6 +31,12 @@ fi
 
 curl -o "$static_dir/pf.css" "$pf_url"
 
+# Patternfly implemented some changes, setting the default height on iframe objects to "auto", which
```

Comment:
Isn't there still some `no-reset` version of patternfly that does not do css reset?

I found `/@patternfly/react-core/dist/styles/base-no-reset.css` in https://www.npmjs.com/package/@patternfly/react-core?activeTab=code. Looking for a better place where to get it.

Reply:
My main concern now is, if some of our layout, grid, etc, rely on the default reset values for our current implementation, then simply removing it could break a lot of our layout and styling. I would prefer to add a task to the next sprints so we can remove and properly check and test if it's ok.

Reply:
Asked in internal Slack at https://redhat-internal.slack.com/archives/C04JMHKSD9C/p1744730368268949, @mcoker informed us that

> it's called "no-globals" now:  https://unpkg.com/@patternfly/patternfly@6.0.0/patternfly-no-globals.css